modules/tab_parameter_estimation/mixture_models/plot_mod_mix_fun.R

```diff
@@ -0,0 +1,61 @@
+plot_mod_mix_fun_ui <- function(id) {
+  ns <- shiny::NS(id)
+
+  r_function(
+    name = "plot_mod",
+    varname = "p_mod_mix",
+    r_function_arg(
+      "p_obj",
+      htmltools::pre(
+        shiny::uiOutput(
+          outputId = ns("p_obj")
+        )
+      )
```

Comment:
We should use uiOutput(container = htmltools::pre). Please add this to the locations where I forgot it as well.

Reply:
To clarify, because it is not visible: the outer pre tag can be omitted and instead be set as argument `container` to uiOutput

Reply:
Got it.